internal/controller/gcpinstall/inputs.go

```diff
@@ -212,6 +212,16 @@ func (ctl *InputsController) Validate(inputs []*wrapinput.Model) bool {
 		}
 
 		switch k {
+		case keyProject:
+			projectID := input.Value()
+			orgID, err := gcp.GetOrganizationID(projectID)
```

Comment:
if there's an error with this or creating the IAM role at the org level, can we fall back to making it at the project level?

Reply:
Leaving as a note for myself (I'll re-request review when it's ready):

To be able to fall back to the project-level, I'll need to create a new environment variable key that does the entire line (`--project=<>` vs. `--organization=<>`)